lib/jazzy/config.rb

```diff
@@ -220,6 +220,11 @@ def expand_path(path)
         SourceDeclaration::AccessControlLevel.from_human_string(acl)
       end
 
+    config_attr :lint,
+      command_line: '--[no-]lint',
+      description: "Generate linter output.",
+      default: false
```

Comment:
OK, that makes more sense.

What’s the difference between `--lint` and the current behavior? Hust that it writes to stdout instead of `undocumented.txt`? That it affects jazzy’s exit code? That is might do checks for things other than undocumented items? Probably all three, right?